docs/topics/signals.rst

```diff
@@ -16,6 +16,36 @@ deliver the arguments that the handler receives.
 You can connect to signals (or send your own) through the
 :ref:`topics-api-signals`.
 
+Here is a simple example showing how you can catch signals and perform some action:
+::
+
+    from scrapy import signals
+
```

Comment:
Hey, @darshanime -- this looks good, only one small thing to adjust before merging: it will be nice to add `from scrapy import Spider` here and a `pass` to the parse method below, so that users don't get an error when trying to run as-is (a quick way to check is to paste the code into a file and run with `scrapy runspider file.py`).

Ready to merge after that. :)